fastai/learner.py

```diff
@@ -383,6 +384,7 @@ def fit_opt_sched(self, phases, cycle_save_name=None, best_save_name=None, use_w
         if len(data_list)==0: data_list = [self.data]
         return fit(self.model, data_list, n_epochs,layer_opt, self.crit,
             metrics=metrics, callbacks=callbacks, reg_fn=self.reg_fn, clip=self.clip, fp16=self.fp16, **kwargs)
-    
 
+    def _determine_loss_func(self, data):
```

Comment:
Rather than raising error here, just set it to some default (mse seems fine). They might set `crit` later so no need for an exception now.

Reply:
Also maybe a better name is `_get_crit` since we normally use `get` prefix in this situation, and use `crit` instead of loss.